app/scripts/views/sign_up.js

```diff
@@ -177,6 +181,14 @@ define(function (require, exports, module) {
         shouldFocusPassword: autofocusEl === 'password',
         showSyncSuggestion: this.isSyncSuggestionEnabled()
       };
+      if (this._webChannelSupportExists()) {
+        context.escapedSyncSuggestionUrl = encodeURI('/signup?service=sync&context=fx_desktop_v3&entrypoint=uitour');
```

Comment:
The `context` query parameter needs to be dynamic based on which browser the user is in as well. That logic to make that decision is also contained in [connect_another_device.js](https://github.com/mozilla/fxa-content-server/pull/4798/files#diff-134857af02a4339fb37cabf9266d6ef0R191).

The `context` query parameter allows FxA to act differently depending on the browser with which FxA is being integrated. For example, users of Firefox for Android should not see the links on the verification page that say "Install Firefox for Android" and "Install Firefox for iOS". 

The logic to make the decision can be extracted from [connect_another_device.js](https://github.com/mozilla/fxa-content-server/pull/4798/files#diff-134857af02a4339fb37cabf9266d6ef0R191) into a location it can be shared. I'm not sure where to best to place this logic, I'll think about that for a little bit and make a suggestion.

We'll also want to change the `entrypoint` query parameter. `entrypoint` is used in our metrics infrastructure to help us track where users originate from. `uitour` has a very specific meaning within our metrics, using it here would alter that meaning. Make `entrypoint` something like `fxa:signup_page`